apps/filecoin-site/src/app/_components/SimpleCard.tsx

```diff
@@ -4,81 +4,69 @@ import { Badge, type BadgeProps } from '@/components/Badge'
 import { CTALink, type CTALinkProps } from '@/components/CTALink'
 import { Heading } from '@/components/Heading'
 
-type SimpleCardProps = {
+export type SimpleCardProps = {
   title: string
   description: string
-  cta: {
+  as?: 'li' | 'div'
```

Comment:
Should be required as discussed previously to ensure we don't use `divs` when using lists, etc.